src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,19 @@
-export const UserInfo = () => {};
+import React from 'react';
+import { User } from '../../types/User';
+
+interface UserInfoProps {
+  user: User;
+}
+
+export const UserInfo: React.FC<UserInfoProps> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
2 attributes can go in one line
```suggestion
  const { name, email } = user;
```

Reply:
thank you, fixed